WordPress/metadata/PlayStoreStrings.po

```diff
@@ -11,21 +11,20 @@ msgstr ""
 "Project-Id-Version: Release Notes & Play Store Descriptions\n"
 
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
-msgctxt "release_note_158"
+msgctxt "release_note_159"
 msgid ""
-"15.8:\n"
-"- Block editor enhancements: Added full-width and wide alignment support for Group, Cover, and Image blocks; added support for the Rounded style in the Image block; and resolved an issue where the toolbar behaved unexpectedly when adding a block.\n"
-"- Reader update: Preview images are now displayed on posts when no featured image is set.\n"
-"\n"
+"15.9:\n"
+"- Block editor: Edit unsupported blocks by enabling Jetpack SSO from the missing block alert. Use the link picker to add URLs to posts. Double-tap text fields in accessibility mode now works as expected.\n"
+"- Improved My Site layout for large-screen devices.\n"
 "\n"
 msgstr ""
 
-msgctxt "release_note_157"
+msgctxt "release_note_158"
 msgid ""
-"15.7:\n"
-"Improvements in the Reader: A new Discover tab you can tailor to your interests and an “(un)follow” button to the site and topic detail screens.\n"
+"15.8:\n"
+"- Block editor enhancements: Added full-width and wide alignment support for Group, Cover, and Image blocks; added support for the Rounded style in the Image block; and resolved an issue where the toolbar behaved unexpectedly when adding a block.\n"
+"- Reader update: Preview images are now displayed on posts when no featured image is set.\n"
 "\n"
-"Block Editor enhancements: A new Pullquote block that lets you add emphasis to a chosen piece of text. Any changes made in menu sliders are now also immediately reflected in block settings.\n"
```

Comment:
Is it expected that `release_note_157` was removed? Like, do we only keep the translations for only the last 2 release versions in GlotPress?

Reply:
Yep, right!